samples/boards/stm32/power_mgmt/blinky/README.rst

```diff
@@ -1,42 +1,39 @@
 .. _stm32-pm-blinky-sample:
 
-Blinky
-######
+STM32 PM Blinky
+###############
 
 Overview
 ********
 
-Blinky is a simple application which blinks an LED forever using the :ref:`GPIO
-API <gpio_api>` in low power context.
+This sample is a minimum application to demonstrate basic power management
+behavior in a basic blinking LED set up using the :ref:`GPIO API <gpio_api>` in
+low power context.
 
 .. _stm32-pm-blinky-sample-requirements:
 
 Requirements
 ************
 
-You will see this error if you try to build Blinky for an unsupported board:
-
-.. code-block:: none
-
-   Unsupported board: led0 devicetree alias is not defined
-
-The board must have an LED connected via a GPIO pin. These are called "User
-LEDs" on many of Zephyr's :ref:`boards`. The LED must be configured using the
-``led0`` :ref:`devicetree <dt-guide>` alias. This is usually done in the
-:ref:`BOARD.dts file <devicetree-in-out-files>` or a :ref:`devicetree overlay
-<set-devicetree-overlays>`.
-
-The board should support enabling PM.
+The board should support enabling PM. For a STM32 based target, it means that
+it should support a clock source alternative to Cortex Systick that can be used
+in core sleep states, as LPTIM ("st,stm32-lptim").
```

Comment:
```suggestion
in core sleep states, as LPTIM (:ref:`dtbinding_st_stm32_lptim`).
```

Reply:
Thanks! I knew there was a way, but couldn't find it!

Reply:
Actually, found out we support 

```
:dtcompatible:`vnd,foo`
```